src/components/ButtonBridge/index.tsx

```diff
@@ -0,0 +1,71 @@
+// This bridge is only meant to cover Chip and Tag components
```

Comment:
Maybe would be better to rename `ButtonBridge` to `Pressable`, I saw this on web and it's a bit more clear what component does

Reply:
I also thought so first. But on the web the pressable is the root of the button, but in our case it is a bridge to the button. We have a `pressable` in rn we use in the button as well. Let me think a way for it